.travis.yml

```diff
@@ -6,6 +6,12 @@ cache:
   - Carthage
   - Inspector/node_modules
 
+before_install:
+- rvm install 2.3.3
```

Comment:
What about using 2.6.2?
The version has `bundler`. We do not need to call `gem install bundler`

Reply:
changed to 2.6.2